client/src/components/AccountForm/index.js

```diff
@@ -1,11 +1,12 @@
 import { h, Component, render } from 'preact';
+import React from 'react';
 import style from './style.css';
 import { validateAccountForm, clearForms } from "../../js/validate-account-form";
-import { LOGIN_PATH, REGISTER_PATH, RESET_PATH } from '../../../config';
+import { BASE_ENDPOINTS } from "../../js/server-requests-utils";
 import linkState from "linkstate";
 import { route } from 'preact-router';
 
-export default class AccountForm extends Component {
+export default class AccountForm extends React.Component {
```

Comment:
what is the reasoning for switching to react component here?

Reply:
The only way to get the tests to run was by making it a React.Component. I think this is an example of why we use preact-compat - there are libraries (used by jest or enzyme) which are tightly coupled to React.

Reply:
Ok I think what I would like to do with this one is merge in the refactoring of path vars to development and then create a new feature branch for preact tests to merge into and explore this further

Reply:
Sounds like a plan.